### DISCUSSION TITLE
Sharing between mac in fullscreen does not display the same resolution

### DISCUSSION THREAD

**Discussion by yene** [rustdesk/rustdesk#8122] [General]
### Bug Description

I run two macbook pros with identicial resolution side by side (1800:1169).
When I share the screen in fullscreen, it is not identical.
In the scaled mode it is small, in the adaptive mode it is blurry, and there are sidebars sometimes.

I expected that both macbook pros next to each other display the same resolution and sharpness.

### How to Reproduce

1. Share screen between two macs that have the same resolution.
2. Compare what is displayed 

### Expected Behavior

Both macs display the same thing

### Operating system(s) on local side and remote side

osx -> osx

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

...

**Comment by rustdesk** [comment]
Please try out https://github.com/rustdesk/rustdesk/releases/tag/nightly, this version support retina screen, it would be clear.

**Comment by yene** [comment]
About the matching layout and black sidebars. 
Should this be a new ticket, where the goal is to have a fullscreen that matches exactly the source display?
